ssh-config

```diff
@@ -0,0 +1,5 @@
+Host bastion
```

Comment:
I wouldn't hardcode the name bastion here. Maybe `bifrost-bastion`? What if I already have a bastion profile on my machine?

Reply:
Renamed to `llm-bastion` to prevent naming clashes. But other than that, for experience ops you can certainly name it whatever you want.